### ISSUE TITLE
Add support for PETODAY Automatic Cat/Dog Feeder

### ISSUE THREAD

**Issue by cbass0290** [0x5e/homebridge-tuya-platform#220]
I was able to successfully link my Tuya "Smart Home" dev project via the plugin in Homebridge. It detected both of my cat feeders, but says that they're unsupported devices, despite indicating "cwwsq" as the model/category in the Tuya IoT portal. Per the instructions, I've attached both logs & redacted the private info (although please let me know if I redacted too much).

Thanks!

[TuyaDeviceList.az164850678965594hUg.log](https://github.com/0x5e/homebridge-tuya-platform/files/10493756/TuyaDeviceList.az164850678965594hUg.log)

[Homebridge_Tuya-MQTT-log.log](https://github.com/0x5e/homebridge-tuya-platform/files/10493748/Homebridge_Tuya-MQTT-log.log)

**Comment by cbass0290** [comment]
I purchased these through Amazon, and they interface with the Tuya Smart app. It looks like they now have a [newer model](https://www.amazon.com/PETODAY-Automatic-Dispenser-Programmable-Customizable/dp/B09BKJXCCQ?th=1) (mine doesn't have the backlit screen).

**Comment by 0x5e** [comment]
I didn't find the suitable homekit service type for pet feeder. If you have any ideas, please let me know.
https://developers.homebridge.io/#/service/

**Comment by cbass0290** [comment]
After a bit of tinkering, I was able to easily achieve what I was looking to do by creating Scenes in the Tuya Smart app.

I created two separate scenes for feeding each cat 1 serving, then set each feeder in the plugin to "hidden" -- The Devices get discovered but hidden, leaving only the Scenes, which come through as Switches.

Now I have two switches in the Home app that allow me to manually feed each cat (using my HomePods as well).

Good enough for me! Thanks for your work on this plugin!
